src/components/camera.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { RNCamera } from 'react-native-camera';
+import colors from '../assets/colors.js'
+
+class Camera extends React.Component {
+  render() {
+    return (
+      <View style={{flex: this.props.flex, flexDirection: 'row'}}>
+        <View style={styles.rowContainer}>
+          <RNCamera
+            ref={ref => {
+              this.camera = ref;
+            }}
+            style = {styles.preview}
+            type={RNCamera.Constants.Type.back}
+            flashMode={RNCamera.Constants.FlashMode.on}
+            permissionDialogTitle={'Permission to use camera'}
+            permissionDialogMessage={'We need your permission to use your camera phone'}
+          />
+          <View style={{flex: 0, flexDirection: 'row', justifyContent: 'center',}}>
```

Comment:
avoid inline style blocks, I see three instances in this file.
It's difficult to be consistent/refactor when we have styles mixed in with the jsx